.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig helps developers define and maintain consistent coding
+# styles between different editors and IDEs.
+# Learn More: http://EditorConfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Yea. Most editors (sublime, atom, etc) have an editorconfig plugin which reads the dot file and enforces these style guidelines. It helps contributors to do things like use spaces instead of tabs without having to configure each project independently.

Sent from my iPhone

> On Mar 20, 2015, at 2:27 PM, Marcus Gartner <notifications@github.com> wrote:
> 
> In .editorconfig:
> 
> > @@ -0,0 +1,12 @@
> > +# EditorConfig helps developers define and maintain consistent coding
> > +# styles between different editors and IDEs.
> > +# Learn More: http://EditorConfig.org
> > +root = true
> > +
> > +[*]
> > +indent_style = space
> > +indent_size = 2
> > +end_of_line = lf
> > +charset = utf-8
> > +trim_trailing_whitespace = true
> > +insert_final_newline = true
> Did you mean to commit this file?
> 
> —
> Reply to this email directly or view it on GitHub.
>